staging/src/k8s.io/kubectl/pkg/cmd/set/set_selector.go

```diff
@@ -164,6 +164,9 @@ func (o *SetSelectorOptions) RunSelector() error {
 	r := o.ResourceFinder.Do()
 
 	return r.Visit(func(info *resource.Info, err error) error {
+		if err != nil {
```

Comment:
This might change behaviour, since we'll break earlier can you double check that?

Reply:
Agree. But I think precheck here makes sense. Similar logic at reconcile https://github.com/kubernetes/kubernetes/blob/5488fe547d0345827d516b81efe4d51762944ddc/pkg/kubectl/cmd/auth/reconcile.go#L192-L197